client-v2/src/components/Editions/ManageEdition.tsx

```diff
@@ -201,13 +185,14 @@ class ManageEdition extends React.Component<
   };
 
   private isIssuePresentForDate = (date: Moment) =>
```

Comment:
So this _would_ have worked before but we didn't have enough data in the state? Is that right?